source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,298 @@
+.. _swupd-3rd-party:
+
+swupd 3rd-party 
+###############
+
+Upstream |CL| offers a plethora of `bundles`_ to choose from.  
+
+To provide you access to even more software outside of the distro, 
+the :ref:`mixer` and :ref:`swupd-guide` tools have added support for 3rd-party
+bundles starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.
+
+And to consume and manage these bundles, a :command:`3rd-party` 
+subcommand has been added to :command:`swupd`.
+
+Follow this guide to see how to build an example 3rd-party bundle, set up a web 
+server to host it, install and manage the bundle on a client system. 
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisite
+*************
+
+* Familiarity with :command:`mixer`
+* Familiarity with :command:`swupd`
+
+Create directory to hold 3rd-party app
+**************************************
+
+#. Create a top-level directory workspace to hold all apps.
+	
+   .. code-block:: bash
+      
+      mkdir ~/my-3rd-party-apps && pushd $_
+
+#. Create a directory for each app and put the content of your software in it.  
+
+   In this example, the `helloclear.sh` app simply prints
+   "Hello Clear!" when invoked.  
+
+   .. code-block:: bash
+
+      # make helloclear directory
+      mkdir -p helloclear/usr/bin && pushd $_
+
+      # create helloclear.sh script
+      cat > helloclear.sh << EOF
+      #!/bin/bash
+      echo "Hello Clear!"
+      EOF
+
+      # make script executable
+      chmod +x helloclear.sh
+
+      popd
+
+Create bundle of 3rd-party app with mixer
+*****************************************
+
+Next, use :command:`mixer` to create a bundle for each of the apps from the previous section.  
+
+#. Install the mixer tool.
+   
+   .. code-block:: bash
+
+      sudo swupd bundle-add mixer
+
+#. Create a mixer workspace.
+   
+   .. code-block:: bash
+
+      mkdir ~/mixer && cd $_
+
+#. Initialize a mix without any default bundles.
+   
+   .. code-block:: bash
+
+      mixer init --no-default-bundles
```

Comment:
Ln 82: When I ran this command, stdout showed this error: 
2020/03/17 22:14:04 ERROR: Failed to retrieve latest published upstream version. Missing proxy configuration? : Get https://cdn.download.clearlinux.org/latest: proxyconnect tcp: EOF

Is this only internal for us? Do you have a workaround flag?

I'm aware this flag isn't likely correct. But if I add it, as shown, it allows me to create a mix.
`mixer init --no-default-bundles --upstream-version 32610`